### ISSUE TITLE
"`LtUnlintNonExistingDefect` Executes Removed Lints Ignoring `.without()` Method"

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#495]
Take a look at this code:

```java
new Program(xmir).without("foo").defects();
```

Here, we removing `foo` lint from the single program lints, and it won't be executed by `Program`. However, `LtUnlintNonExistingDefect` does not know about that, and executes `foo` lint inside:

https://github.com/objectionary/lints/blob/a3bc30519bba7711f9152e28c6fab08fde13c65c/src/main/java/org/eolang/lints/LtUnlintNonExistingDefect.java#L120-L135

This is wrong, if some lint was removed, then it should be removed from `LtUnlintNonExistingDefect` too.

P.S.: The same story in WPA scope.

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel definitely a bug, should be fixed

**Comment by 0crat** [comment]
@h1alexbel Great catch on that bug! As per our policy, you've earned **8** points for your sharp eye. Your total balance now stands at [+133](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep up the excellent work in enhancing our project's quality. Remember, if you spot anything else that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) quite right, [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your contributions are invaluable!
